Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -78,6 +79,8 @@
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.concurrent.atomic.AtomicReferenceArray;
```

Comment:
Hint: to avoid unused imports and other typos I always review the code before commit. I press Ctrl+D (Cmd+D) in Local Changes tab right before commit or right in the commit dialog and browse all changes by simply pressing F7. Editing and reformatting code works in the diff view.